packages/pyright-internal/src/analyzer/textEditUtils.ts

```diff
@@ -0,0 +1,24 @@
+import { Range, TextDocumentContentChangeEvent } from 'vscode-languageserver';
```

Comment:
This file is missing a header comment.

Reply:
I realized I'm not using this anymore. I just deleted the file.